sqlx-core/src/postgres/types/chrono.rs

```diff
@@ -235,6 +236,38 @@ fn postgres_epoch() -> DateTime<Utc> {
     Utc.ymd(2000, 1, 1).and_hms(0, 0, 0)
 }
 
+impl TryInto<Duration> for PgInterval {
```

Comment:
I'm reasonably sure that you want to `impl TryFrom<PgInterval> for Duration` instead.